shaders_js.go

```diff
@@ -0,0 +1,83 @@
+//+build netgo
+
+package engo
+
+import (
+	"github.com/gopherjs/gopherjs/js"
+)
+
+type Shader interface {
+	Initialize(width, height float32)
+	Pre()
+	Draw(texture *js.Object, buffer *js.Object, x, y, rotation float32)
```

Comment:
I tried before and got this: pong.js:1 engo/render.go:259:25: cannot use e.RenderComponent.drawable.Texture() (value of type *github.com/gopherjs/gopherjs/js.Object) as *engo.io/gl.Texture value in argument to rs.currentShader.Draw

Maybe  I did it wrong?